adapters/playwire_ortb/params_test.go

```diff
@@ -0,0 +1,52 @@
+package playwire_ortb
+
+import (
+	"encoding/json"
+	"github.com/prebid/prebid-server/openrtb_ext"
+	"testing"
+)
```

Comment:
Totally nitpicky, but might be worth switching to a 4-space indent to match the rest of the Go code in the repo.